salt/metalk8s/metalk8s-ui/deployed.sls

```diff
@@ -1,5 +1,20 @@
 {% set kubeconfig = "/etc/kubernetes/admin.conf" %}
```

Comment:
We should rename the `metalk8s-ui` folder into `ui` since we are already in the `metalk8s` tree

Reply:
yes good point. We change it later ;)